pkg/plugins/plugins.go

```diff
@@ -73,33 +77,40 @@ func (pm *PluginManager) Init() error {
 	}
 
 	pm.log.Info("Starting plugin search")
+
+	pm.log.Info("Scanning main plugin directory")
 	plugDir := path.Join(setting.StaticRootPath, "app/plugins")
-	if err := pm.scan(plugDir); err != nil {
-		return errutil.Wrapf(err, "Failed to scan main plugin directory '%s'", plugDir)
+	if err := pm.scan(plugDir, false); err != nil {
+		return errutil.Wrapf(err, "failed to scan main plugin directory '%s'", plugDir)
 	}
 
-	pm.log.Info("Checking Bundled Plugins")
+	pm.log.Info("Scanning bundled plugins directory")
```

Comment:
Move `plugDir = path.Join(setting.HomePath, "plugins-bundled")` before this and include in log?

```suggestion
	pm.log.debug("Scanning bundled plugins directory", "dir", plugDir)
```